app/lib/shared/memcache.dart

```diff
@@ -30,13 +30,13 @@ const String searchServiceResultPrefix = 'search_service_result_';
 // Keeping it under that limit in order to offset char coding or other payloads.
 const _contentLimit = 1000 * 1000;
 
-class SimpleMemcache {
+class SimpleMemcache2 {
```

Comment:
??

Reply:
oh, left over, I probably meant to delete this file completely :)